account/guides/setupRegistrar.md

```diff
@@ -0,0 +1,147 @@
+# Setup Your Dashboard Account For Registrar {#top}
+This guide walks through the initial setup for Bandwidth's Registrar. The Registrar allows you to use Bandwidth to create phone calls to and from our network using registered devices.
+
+You must contact [Bandwidth Customer Support](http://support.bandwidth.com/) to get your SIP domain and port number.
+
+## Pre-Reqs {#pre-requs}
+* [Twilio Account](http://twilio.com/)
+* [Bandwidth Dashboard Account](http://bandwidth.com/)
+* Registrar activated: [_contact sales_](https://www.bandwidth.com/)
+* Your default port ex: `5006` [_contact support_](http://support.bandwidth.com/)
+
+## Steps {#steps}
+1. [Create New Realm](#create-new-realm)
+1. [Create New SIP Credentials](#create-new-sip-credentials)
+1. [Register Device](#register-sip-device)   
+1. [Create call via Twilio with new SIP Creds](#create-call)
+
+## Create New Realm {#create-new-realm}
+In order to route the calls through Bandwidth, you'll need to create a new realm.  Upon realm creation, it will be used as part of the `Realm` combined with `<randomAccountHex>.auth.bandwidth.com`.
+
+| Value      | Required | Description |
+|------------|----------|-------------|
+| `Realm` | required | String identifying the realm. |
+| `Description`   | optional | A string used for a description of the realm |
+
+## Create New Sip Credentials {#create-new-sip-credentials}
+In order to route the outbound calls through Bandwidth, you'll need to create a new set of SIP credentials. To generate your own hash values, you can follow the instructions below, but if you would like Bandwidth to generate those hash values for you, you can skip ahead to the [Using the UI](#using-the-ui) section of this guide.
+
+| Value      | Required | Description |
+|------------|----------|-------------|
+| `Username` | required | String identifying the user. |
+| `Domain`   | optional | String refining the identity of the user. The Domain will be joined to the UserName with an @ to create a composite username. For example, the UserName bob could be combined with the domain somewhere.com to create a composite username bob@somewhere.com |
```

Comment:
```suggestion
| `Domain`   | optional | String defining the identity of the user. The Domain will be joined to the UserName with an @ to create a composite username. For example, the UserName bob could be combined with the domain somewhere.com to create a composite username bob@somewhere.com |
```
This should also be fixed in the Network Bridge guide too

Reply:
Fixed!